gaming/minecraft/en.md

```diff
@@ -31,7 +25,33 @@ sudo tar xf data.tar.xz -C /
 sudo rm control.tar.gz data.tar.xz debian-binary Minecraft.deb
 ```
 
-Integrate the files into your system:
+#### Now edit the desktop file to integrate openjdk-8 environment:
+
+On Solus Budgie or Gnome:
+
+``` bash
+sudo gedit /usr/share/applications/minecraft-launcher.desktop
+```
+
+On Solus Mate:
+
+``` bash
+sudo pluma /usr/share/applications/minecraft-launcher.desktop
+```
+
+On Solus Plasma:
+
+``` bash
+kate /usr/share/applications/minecraft-launcher.desktop
```

Comment:
Shouldn't this also be sudo or does kate do some sort of fancy polkit permissions prompt?